chainer_chemistry/datasets/tox21.py

```diff
@@ -73,11 +73,14 @@ def postprocess_label(label_list):
                            labels=labels)
 
     if return_smiles:
-        train_result = parser.parse(get_tox21_filepath('train'))
+        train_result = parser.parse(get_tox21_filepath('train'),\
+                                    return_smiles=True)
```

Comment:
Can you move  `parser.parse(..., return_smiles=return_smiles)` outside of `if return_smiles ... else ... ` clause?
Other `val_result, test_result` as well.

Reply:
OK